entities/OrderDetails.java

```diff
@@ -0,0 +1,72 @@
+package entities;
+
+import constants.Status;
+
+import java.time.Instant;
+
+public class OrderDetails extends Base {
+
+    //Attributes
+    private Integer orderID;
```

Comment:
Instead refer to the whole `Order` entity

Reply:
done